crates/ty_python_semantic/resources/mdtest/call/replace.md

```diff
@@ -17,7 +17,8 @@ from datetime import time
 t = time(12, 0, 0)
 t = replace(t, minute=30)
 
-reveal_type(t)  # revealed: time
+# TODO: this should be `time`, once we support specialization of generic protocols
+reveal_type(t)  # revealed: Unknown
```

Comment:
https://github.com/python/typeshed/pull/14786 changed the signature of `copy.replace` from
```py
_SR = TypeVar("_SR", bound=_SupportsReplace)

def replace(obj: _SR, /, **changes: Any) -> _SR: ...
```
to
```py
def replace(obj: _SupportsReplace[_RT_co], /, **changes: Any) -> _RT_co: ...
```
which our generics solver does not support yet :cry:.

I don't think it's important enough to implement a workaround for, though.